pillar.example

```diff
@@ -1,6 +1,31 @@
 mongodb:
   lookup:
```

Comment:
Make this `overrides` as we discussed to maintain consistency. At some point we should go back through our other formulas and do the same. (don't forget to change it in the map file too)